src/python/pants/backend/python/target_types.py

```diff
@@ -124,11 +127,24 @@ class PexEntryPointField(StringField, AsyncFieldMixin):
         1) 'app.py', Pants will convert into the module `path.to.app`;
         2) 'app.py:func', Pants will convert into `path.to.app:func`.
 
+    You must use the file name shorthand for file arguments and `--changed-since` to work with this
+    target.
+
     To leave off an entry point, set to '<none>'.
     """
 
     alias = "entry_point"
 
+    @property
+    def filespec(self) -> Filespec:
+        if not self.value:
```

Comment:
Can this be `None`? I thought it had to be a string (even if the string `<none>`)? I notice that you don't check this in the corresponding awslambda side.

Reply:
Yeah, we don't yet require `entry_point` to be `required = True` because of the shorthand where `sources` is set instead. That will be removed soon, and we can align with aws lambda.